ninja.lua

```diff
@@ -204,6 +208,8 @@ function ninja.generateWorkspace(wks)
 				table.insert(args, arg);
 			end
 		end
+		table.sort(args)
```

Comment:
premake overrides args with latest in case of duplicated entries (i.e `premake5 --cc=gcc --cc=clang` uses clang, whereas `premake5 --cc=clang --cc=gcc` uses gcc).
NOt sure what you want to fix with that.